### PR TITLE
[15]Research Programming Languages

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Researched on Programming Languages
- `Created a folder and added a file with link to shared document which has answers to questions

### CODE REVIEW

File: submissions/chabuChabu/css/contact-styles.css [hunk 1]
```diff
@@ -23,6 +23,10 @@ header .logo{
 	height: inherit;
 	margin-left: 20px;
 }
+<<<<<<< HEAD
```
Comment (adaugochi) [inline submissions/chabuChabu/css/contact-styles.css:26]: You have a lot on unresolved merge conflict. Locate the session of your file and remove all this `<<<< HEAD ==== >>>> 206...` Same also in other places

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@ChabuChabu Fix the merge conflict in this pull request.

File: submissions/chabuChabu/css/contact-styles.css (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/css/contact-styles.css]: Changes made

**Comment by ChabuChabu** [comment]
> @ChabuChabu Fix the merge conflict in this pull request.

Changes made

**Review by ChabuChabu — COMMENTED** [review]
Conflicts addressed

**Review by adaugochi — APPROVED** [review]

**Pull request merged**